server/services/workAndSkillsPageService.ts

```diff
@@ -157,7 +203,7 @@ export default class WorkAndSkillsPageService {
     return multiListArray
   }
 
-  async getLearnerGoals(prisonerNumber: string, curiousApiClient: CuriousApiClient) {
+  private async getLearnerGoals(prisonerNumber: string, curiousApiClient: CuriousApiClient): Promise<LearnerGoalsData> {
```

Comment:
Corrected / strengthened the visibility of this function whilst I was here 👍 (nothing calls it outside of this class, so no need for it to be effectively `public`)